frontend/taipy-gui/src/components/Taipy/TimeSelector.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021-2024 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+import React, { useState, useMemo } from "react";
+import Box from "@mui/material/Box";
+import Tooltip from "@mui/material/Tooltip";
+import { ErrorBoundary } from "react-error-boundary";
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
+
+import { MobileTimePicker } from '@mui/x-date-pickers/MobileTimePicker';
+import { getSuffixedClassNames, TaipyActiveProps, TaipyChangeProps, getCssSize } from "./utils";
+import { getDateTime } from "../../utils";
+import { useClassNames, useDynamicProperty, useFormatConfig } from "../../utils/hooks";
+import ErrorFallback from "../../utils/ErrorBoundary";
+import Field from "./Field";
+
+
+
+interface TimeSelectorProps extends TaipyActiveProps, TaipyChangeProps {
+  as_clock?: boolean;
```

Comment:
as_clock is transformed into asClock automatically

Reply:
Changed, done